obs-packaging/qemu-vanilla/_service-template

```diff
@@ -2,7 +2,7 @@
 <services>
  <service name="tar_scm">
     <param name="scm">git</param>
-    <param name="url">https://github.com/kata-containers/qemu.git</param>
+    <param name="url">https://github.com/qemu/qemu.git</param>
```

Comment:
do you know if the `checkout` to the version 2.11 is automatic?

Reply:
kind of the checkout is don by the xml node.
` <param name="revision">@QEMU_VANILLA_HASH@</param>`

Reply:
make sense, thanks